internal/rootcoord/root_coord.go

```diff
@@ -2259,7 +2261,8 @@ func (c *Core) isValidRole(entity *milvuspb.RoleEntity) error {
 		return errors.New("the name in the role entity is empty")
 	}
 	if _, err := c.meta.SelectRole(util.DefaultTenant, &milvuspb.RoleEntity{Name: entity.Name}, false); err != nil {
-		return err
+		log.Error("fail to select the role", zap.Error(err))
```

Comment:
Warn or info

Reply:
The request will return the err, use warn?

Reply:
OK， warn sounds good

Reply:
Have updated